index.md

```diff
@@ -1,30 +1,32 @@
-### Introduction
+### OpenTK
 
-The Open Toolkit is set of fast, low-level C# bindings for OpenGL, OpenGL ES and OpenAL. It runs on all major platforms and powers hundreds of apps, games and scientific research.
+The Open Toolkit is set of fast, portable, low-level C# bindings for OpenGL, OpenGL ES, OpenAL, and OpenCL.  It runs on all major platforms and powers hundreds of apps, games, and scientific research programs.
 
 OpenTK provides several utility libraries, including a math/linear algebra package, a windowing system, and input handling.
 
-### Developer decisions
+OpenTK is not a game engine, nor does it try to be one:  It offers a useful set of tools and lets you decide how to use them.  It is the ideal starting point if you want to write a game or scientific application from the ground up.
 
-OpenTK is not a game engine, nor does it try to be one. It offers a useful set of libraries, and leaves it up to the developer to make the decisions on how to use them.
+### Current Status
 
-It is the ideal starting point if you want to write a game or scientific application from the ground up.
+- **OpenTK 4.x** is the current, stable version.  It targets .NET Core 3.x and .NET 5+.
+   - [OpenTK 4.x](https://www.nuget.org/packages/OpenTK/) [![NuGet](https://img.shields.io/nuget/v/OpenTK.svg)](https://www.nuget.org/packages/OpenTK/) [![NuGet](https://img.shields.io/nuget/dt/OpenTK.svg)](https://www.nuget.org/packages/OpenTK/)
+   - [OpenTK 4.x GLWpfControl](https://www.nuget.org/packages/OpenTK.GLWpfControl/) (beta WPF control)
+   - [OpenTK 4.x GLControl](https://www.nuget.org/packages/OpenTK.WinForms) (beta WinForms control)
+- **OpenTK 3.x** is the legacy version.  It targets .NET Framework 4.x.
+   - [OpenTK 3.x](https://www.nuget.org/packages/OpenTK/3.3.1)
+   - [OpenTK GLControl 3.x](https://www.nuget.org/packages/OpenTK.GLControl/) [![NuGet](https://img.shields.io/nuget/v/OpenTK.GLControl.svg)](https://www.nuget.org/packages/OpenTK.GLControl/) [![NuGet](https://img.shields.io/nuget/dt/OpenTK.GLControl.svg)](https://www.nuget.org/packages/OpenTK.GLControl/)
+- OpenTK 5.0, the next major revision of OpenTK, [is in development](https://github.com/opentk/opentk/issues/1207) and is likely to be released later in 2021.  We would welcome any [contributions](https://github.com/opentk/opentk/)!
 
-### Project Status (25/10/2018)
-Work is well underway on OpenTK 4.0, which targets .NET Standard/Core. We would welcome any [contributions](https://github.com/opentk/opentk/)!
+### Help and Support
 
+If you have a question or want to know how to get started, check our extensive [FAQ](faq.html), or try [our tutorials](https://opentk.net/learn/index.html).
```

Comment:
The tutorial link goes to the website tutorials which unfortunately are still OpenTK 3 tutorials, linking to https://github.com/opentk/LearnOpenTK and explaining a bit of the tutorial situation is probably a good idea to do here.

Reply:
I've added updates to the tutorials links, so that both are included.